docs/tutorials/detached-mode/save-load-checkpoints.rst

```diff
@@ -0,0 +1,125 @@
+.. _save-load-checkpoints:
+
+###########################################
+ Save and Load the State Using Checkpoints
+###########################################
+
+.. meta::
+   :description: Learn how to utilize detached mode to save and load states via checkpoints. This guide will lead you through the necessary steps.
+
+Leveraging :ref:`detached mode <detached-mode-index>`, you can easily save the state at a particular
+point during training and restore it when needed. This is especially useful for resuming training
+after interruptions or failures.
+
+For the full script, visit the `Github repository
+<https://github.com/determined-ai/determined/blob/main/examples/features/unmanaged/2_checkpoints.py>`_.
+
+************
+ Objectives
+************
+
+These step-by-step instructions walk you through:
+
+-  Initializing the core context with checkpoint storage
+-  Loading the latest checkpoint
+-  Resuming sending metrics to the trial
+-  Saving checkpoints periodically
+
+After completing this guide, you will be able to:
+
+-  Understand how checkpoints operate in detached mode
+-  Implement state-saving and restoration in your training routine
+-  Use the Core API to handle checkpoints effectively
+
+***************
+ Prerequisites
+***************
+
+**Required**
+
+-  A Determined cluster
+
+**Recommended**
+
+-  :ref:`simple-metrics-reporting`
+
+*************************************************************
+ Step 1: Initialize the Core Context with Checkpoint Storage
+*************************************************************
+
+To begin, you need to set up the core context, specifying the checkpoint storage path. If recovering
+from a failure, an external experiment and trial id can be used to identify which artifact to log
+metrics to:
+
+.. code:: python
+
+   def main():
+       core_v2.init(
+           defaults=core_v2.DefaultConfig(
+               name="unmanaged-2-checkpoints",
+               checkpoint_storage="/path/to/checkpoint",
+           ),
+           unmanaged=core_v2.UnmanagedConfig(
+               external_experiment_id="my-existing-experiment",
+               external_trial_id="my-existing-trial",
+           )
```

Comment:
```suggestion
           ),
```

Reply:
@dzhu are you sure this change should be made? The , separates the two arguments 'defaults' and 'unmanaged'

Reply:
I'm not saying to delete that comma, which is a few lines up;
 I'm saying to add a new one.

Reply:
Whoops got my wires crossed, my bad